APMrover2/Parameters.cpp

```diff
@@ -603,6 +603,14 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Path: ../libraries/AC_Avoidance/AP_OAPathPlanner.cpp
     AP_SUBGROUPINFO(oa, "OA_", 45, ParametersG2, AP_OAPathPlanner),
 
+    // @Param: R_OPTIONS
+    // @DisplayName: Rover Options
+    // @Description: Bitmask to enable Rover options:
+    // @Values: 0:None,1:FS allowed in HOLD mode, 2:Enable loiter at end of RTLs
+    // @Bitmask: 0:Enable FS in HOLD mode, 1:Enable loiter at end of RTLs
+    // @User: Advanced
+    AP_GROUPINFO("R_OPTIONS", 46, ParametersG2, r_options, 0),
```

Comment:
This should probably not be prefixed with `R_`

Reply:
changed per devcall